### PR TITLE
docs: fix manpage for containers-storage.conf

### PR DESCRIPTION
**Pull request by Silvanoc**
Small fixes in the manpage for `containers-storage.conf`.

Signed-off-by: Silvano Cirujano Cuesta <silvano.cirujano-cuesta@siemens.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@giuseppe @nalind @vrothberg @TomSweeneyRedHat PTAL

File: docs/containers-storage.conf.5.md [hunk 1]
```diff
@@ -93,10 +93,10 @@ until all of the entries have been used for maps.
       remap-user = "storage"
       remap-group = "storage"
 
-ostree_repo = ""
-   If specified, use OSTree to deduplicate files with the overlay backend.
+**ostree_repo** = ""
+   If specified, use OSTree to deduplicate files with the overlay or vfs backends.
 
-skip_mount_home = "false"
```
Comment (vrothberg) [inline docs/containers-storage.conf.5.md]: Let's keep the default value. Currently, it looks like a string but only "true" or "false" are valid values.

Comment (Silvanoc) [reply docs/containers-storage.conf.5.md]: > Let's keep the default value. Currently, it looks like a string but only "true" or "false" are valid values.

Done

**Comment by Silvanoc** [comment]
> Other than @vrothberg 's nit, LGTM.
> If you've the time, could you also alphabetize all the options in the "Storage Options Table" and also the "Storage Table" sections?

Done

File: docs/containers-storage.conf.5.md [hunk 2]
```diff
@@ -93,13 +100,6 @@ until all of the entries have been used for maps.
...
-ostree_repo = ""
-   If specified, use OSTree to deduplicate files with the overlay backend.
-
-skip_mount_home = "false"
-   Set to skip a PRIVATE bind mount on the storage home directory.
-Only supported by certain container storage drivers (overlay).
-
 [storage.options.thinpool]
 
 Storage Options for thinpool
```
Comment (TomSweeneyRedHat) [inline docs/containers-storage.conf.5.md]: I know it's not yours, but can you remove line 103 and then add three pounds  signs in front of 105?

'### Storage Options for thinpool`

If you don't want to deal with it, no worries, I'll follow up and fix it.

**Comment by TomSweeneyRedHat** [comment]
LGTM
One possible old thing to fix.

**Comment by TomSweeneyRedHat** [comment]
and more importantly @Silvanoc thanks for taking this on and the work to date!

File: docs/containers-storage.conf.5.md (hunk 2 above)
Comment (Silvanoc) [reply docs/containers-storage.conf.5.md]: > If you don't want to deal with it, no worries, I'll follow up and fix it.

If I'm on it, I can also tie up loose ends.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**